sample/SampleAspNetCoreApp/Startup.cs

```diff
@@ -14,15 +14,20 @@ public class Startup
 	{
 		public Startup(IConfiguration configuration) => Configuration = configuration;
 
-		public IConfiguration Configuration { get; }
-
-		// This method gets called by the runtime. Use this method to add services to the container.
+		public IConfiguration Configuration { get; }
+
+		// This method gets called by the runtime. Use this method to add services to the container.
 		public void ConfigureServices(IServiceCollection services)
 		{
 			ConfigureServicesExceptMvc(services);
 
-			services.AddMvc()
-				.SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+#if NETCOREAPP3_0
+			services.AddRazorPages().SetCompatibilityVersion(CompatibilityVersion.Version_3_0);

```

Comment:
I experimented with Razor Pages issue and add it according to [the documentation](according to the documentation). However, as it was described in [other section](https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#mvc-service-registration) `AddMvc()` continues to behave as it has in previous releases.
It seems that I forgot to change it back after my investigations

Reply:
Thanks for the info. Sounds ok. I think we should then use mvc everywhere, so people don't wonder why we changed and in case of a new version what they should add here.